lib/internal/webstreams/readablestream.js

```diff
@@ -987,13 +992,29 @@ class ReadableStreamBYOBReader {
 
     // Supposed to assert here that the view's buffer is not
     // detached, but there's no API available to use to check that.
+
+    const min = options?.min ?? 1;
+    if (typeof min !== 'number')
+      return PromiseReject(new ERR_INVALID_ARG_TYPE('options.min', 'number', min));
+    if (!NumberIsInteger(min))
+      return PromiseReject(new ERR_INVALID_ARG_VALUE('options.min', min, 'must be an integer'));
+    if (min <= 0)
+      return PromiseReject(new ERR_INVALID_ARG_VALUE('options.min', min, 'must be greater than 0'));
```

Comment:
I would have liked to use [validateInteger](https://github.com/nodejs/node/blob/main/lib/internal/validators.js#L94-L99) for this part, but:
* `validateInteger` *throws*, whereas `read(view)` must *reject*
* `validateInteger` throws `RangeError`s, whereas `read(view)` must throw `TypeError`s for these checks

So I guess I have to copy-paste... 🤷‍♂️

Reply:
You can extract it to a common method that returns an err and a throwing version that throws it (or rejects it) but I think this copy-paste is fine here as it's readable and tested. If it bothers you you can make `read` an async function which would cause throws to become rejects and it doesn't throw synchronously anyway